integration_test/integration_tests/src/integration_tests/test_integration.py

```diff
@@ -0,0 +1,178 @@
+import contextlib
+import os
+import threading
+import unittest
+from datetime import timedelta
+
+from omotes_sdk.config import RabbitMQConfig
+from omotes_sdk.job import Job
+from omotes_sdk.omotes_interface import OmotesInterface
+from omotes_sdk.types import ParamsDict
+from omotes_sdk_protocol.job_pb2 import JobProgressUpdate, JobStatusUpdate, JobResult
+import psycopg
+
+from integration_tests.job_submitter import run_high_throughput_test
+
+
+RABBITMQ_CONFIG = RabbitMQConfig(
+    username=os.environ["RABBITMQ_OMOTES_USER_NAME"],
+    password=os.environ["RABBITMQ_OMOTES_USER_PASSWORD"],
+    virtual_host=os.environ["RABBITMQ_VIRTUALHOST"],
+    host=os.environ["RABBITMQ_HOST"],
+    port=int(os.environ["RABBITMQ_PORT"]),
+)
+
+SQL_CONFIG = {
+    "host": os.environ["POSTGRESQL_HOST"],
+    "port": os.environ["POSTGRESQL_PORT"],
+    "database": os.environ["POSTGRESQL_DATABASE"],
+    "username": os.environ["POSTGRESQL_USERNAME"],
+    "password": os.environ["POSTGRESQL_PASSWORD"],
+}
+
+
+class omotes_client:
```

Comment:
If one test fails and takes the omotes_client with it, it will cause failures on other tests which is why I thought it would make sense to keep them separate for clarity-sake in case a test fails.

Reply:
Ok, makes sense.  
Better to have an overview of which tests fail instead of going through them one by one.